css/common.css

```diff
@@ -0,0 +1,166 @@
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+
+body {
+    font-family: "Roboto", sans-serif;
+    -webkit-font-smoothing: antialiased;
+}
+
+
+/* Header */
+
+header {
+    padding: 0 5rem;
+    justify-content: space-between;
+    background: rgba(196, 196, 196, 0.1);
```

Comment:
I would try to use CSS variables instead of hard-coding colors, It will make it very easy if you need to change the color in the future. 
You can declare variables like so: 
 :root {
    --grey-dark: #848993;
    --grey-darker: #4D5057;
    --grey-light: #E5E5E5;
    --orange-dark: #E0633A;
    --orange-light: #FCEFEB;
    --white: #fff;
    --white-transparent: rgba(255,255,255,0.9); 
}
and use them like this: 
color: var(--white);

Reply:
Great idea!

Reply:
I've started using variables on my own work to try them out, I will definitely start using it on everything I do. Thank you.